plugins/inputs/phpfpm/phpfpm.go

```diff
@@ -46,14 +47,25 @@ var sampleConfig = `
   #   - http: the URL must start with http:// or https://, ex:
   #       "http://localhost/status"
   #       "http://192.168.130.1/status?full"
+	#
   #   - unixsocket: path to fpm socket, ex:
   #       "/var/run/php5-fpm.sock"
   #       "192.168.10.10:/var/run/php5-fpm-www2.sock"
+	#       or using a custom fpm status path
+	#       "/var/run/php5-fpm.sock:fpm-custom-status-path"
+	#       or custom host, and custom fpm status path
+	#       "192.168.10.10:/var/run/php5-fpm-www2.sock:fpm-custom-status-path"
+	#
+	#        Note that the unixsocket only works when telegraf run on the same
+	#        host which is being gather metric because unix socket require local
+	#        file access. The IP address field is used as a tag, it doesn't mean
```

Comment:
@sparrc Removed that tag feature. I also rebase it with master so it's ready to merge if thing looks good.